### PR TITLE
Design - In advance receipt update, provide get all adjusted invoices

### PR DESCRIPTION
**Pull request by sadik-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 1]
```diff
@@ -689,6 +689,42 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 placeholder="Description"></textarea>
                         </div>
                     </div>
+
+                    <ng-template>
+                        <div class="clearfix adjust-receipt font-14">
+                            <p class="bold">Adjusted Receipts <a href="javascript:void(0)"><i class="icon-edit11 font-10"></i></a></p>
+                            <p class="font-12 text-light mb-1">₹2400.00</p>
+                            <div class="clearfix adjustment-filed">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: Use `adjustment-field` in class name

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 2]
```diff
@@ -689,6 +689,42 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                                    </div>
+
+                                    <div class="col-sm-2 pl-05">
+                                        <div class="form-group">
+                                            <label>Tax</label>
+                                            <input type="text" class="form-control mb-1" value="₹150.00" disabled>
+                                            <input type="text" class="form-control mb-1" value="₹150.00" disabled>
+                                        </div>
+                                    </div>
+                                </div>monthpicker
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: Remove monthpicker

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss [hunk 3]
```diff
@@ -190,6 +190,33 @@ span.eligible-ITC {
         margin-bottom: 4px;
     }
 }
+.adjust-receipt {
+    padding: 10px;
+    border: 1px solid #D9D9D9;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss]: Use SASS variables for colors

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 4]
```diff
@@ -5586,6 +5586,15 @@ datepicker-inner .well table tbody tr td .btn-default:hover span {
 datepicker-inner .well table tbody tr td .btn-default:hover .text-success {
     color: #fff;
 }
+datepicker-inner .well table tbody tr td button.btn.btn-default.btn-info .text-success {
+    color: #3C763F !important;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss]: Use SASS variables for colors

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 5]
```diff
@@ -5586,6 +5586,15 @@ datepicker-inner .well table tbody tr td .btn-default:hover span {
...
+}
+datepicker-inner .well table tbody tr td button.btn.btn-default.btn-info:hover .text-success, datepicker-inner .well table tbody tr td button.btn.btn-default.btn-link:hover .text-success {
+    color: #fff !important;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss]: Use SASS variables for colors

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html (hunk 1 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: class name changed  adjustment-filed to adjustment-field

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html (hunk 2 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: Removed monthpicker

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss (hunk 3 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss]: now I used sass var colors

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 5 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: now I used sass var colors

**Pull request merged**